HarmonyCore/FileIO/FileIOErrorHelper.dbl

```diff
@@ -15,6 +15,8 @@ namespace Harmony.Core.FileIO
 		proc
 			if(result == FileAccessResults.RecordDataNotSame) then
 				throw new RecordNotSameException()
+			else if (result == FileAccessResults.TimedOut) then
```

Comment:
does this return code get returned anywhere?

Reply:
Removed enum member. See 2e9806cb8c384b82ef3effd1c36ab5b09006dc02